code/src/components/TaskSummery.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import { useSelector } from "react-redux";
+
+import styled from "styled-components";
+
+export const TaskSummery = () => {
+  // Get list information for this task-list:
+  const list = useSelector((store) => store.tasks.list);
```

Comment:
Same here - since your only using the items-array you can get that instead:
`const items = useSelector(store => store.tasks.list.items)`